packaging/installer/methods/kubernetes.md

```diff
@@ -0,0 +1,134 @@
+<!--
+title: "Install Netdata on a Kubernetes cluster"
+description: "Use Netdata's Helm chart to bootstrap a Netdata monitoring and troubleshooting toolkit on your Kubernetes (k8s) cluster."
+custom_edit_url: https://github.com/netdata/netdata/edit/master/packaging/installer/methods/kubernetes.md
+-->
+
+# Install Netdata on a Kubernetes cluster
+
+This document details how to install Netdata on an existing Kubernetes (k8s) cluster. By following these directions, you
+will use Netdata's [Helm chart](https://github.com/netdata/helmchart) to bootstrap a Netdata deployment on your cluster.
+The Helm chart installs one parent pod for storing metrics and managing alarm notifications plus an additional child pod
+for every node in the cluster.
+
+Each child pod will collect metrics from the node it runs on in addition to [22 supported
+services](https://github.com/netdata/helmchart#service-discovery-and-supported-services) via [service
+discovery](https://github.com/netdata/agent-service-discovery/). Each child pod will also collect
+[cgroups](/collectors/cgroups.plugin/README.md),
+[Kubelet](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/k8s_kubelet), and
+[kube-proxy](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/k8s_kubeproxy) metrics from its node.
+
+To install Netdata on a Kubernetes cluster, you need:
+
+-   A working cluster running Kubernetes v1.9 or newer.
+-   The [Helm package manager](https://helm.sh/) on an administrative system.
+-   The [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) command line tool on the same system.
+
+The default configuration creates four pods in your cluster: three `child` pods and one `parent` pod. The `child` pods
+collect metrics and stream the information to the `parent` pod, which uses two persistent volumes to store metrics and
+alarms. The `parent` pod also handles alarm notifications and enables the Netdata dashboard using an ingress controller.
+
+## Install the Netdata Helm chart
+
+Download the [Netdata Helm chart](https://github.com/netdata/helmchart) on the administative system where you have the
```

Comment:
Preferred way to install charts is via repository and we have one: https://netdata.github.io/helmchart/

To install the repo:

```
helm repo add netdata https://netdata.github.io/helmchart/
helm repo update
```

To install the chart:

```
helm install netdata netdata/netdata
```

Reply:
Note: repo is a bit outdated, added an issue to bring it up to date: https://github.com/netdata/infra/issues/626

Reply:
Note: repo is a bit outdated, added an issue to bring it up to date

Reply:
Cool! I didn't know that was a thing. Updated this portion.

Reply:
Sorry @joelhans , seems like I spoke too soon. The repo isn't official yet, so ignore my comments here for now.

Reply:
No worries! Reverted back to the old method.